src/components/HURUmap/Panel/MobilePanel/index.js

```diff
@@ -0,0 +1,66 @@
+import { Typography } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Panel from "./Panel";
+import useStyles from "./useStyles";
+
+import { ReactComponent as TopIcon } from "@/pesayetu/assets/icons/Component 130 – 1.svg";
+import Tabs from "@/pesayetu/components/Tabs";
+import slugify from "@/pesayetu/utils/slugify";
+
+function MobileTabPanel({ dataItems, geography, activeType, ...props }) {
+  const classes = useStyles(props);
+  const activeTab = Math.max(
+    dataItems.findIndex(({ title }) => title === activeType),
+    0
+  );
+  const formatedItems = dataItems.map((item) => {
+    return {
+      label: item.title,
+      href: `#${item.title}`,
+      children: <Panel item={item} geography={geography} {...props} />,
+    };
+  });
+  const scrollToTop = () => {
+    document
+      .getElementById(slugify(geography.name))
+      .scrollIntoView({ behavior: "smooth" });
+  };
```

Comment:
You've never heard of `window.scrollTo`? As in `window.scrollTo({ top: 0, behavior: 'smooth' });`